src/main/java/com/ticketbookingapp/domain/ticket/Ticket.java

```diff
@@ -0,0 +1,28 @@
+package com.ticketbookingapp.domain.ticket;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.math.BigDecimal;
+
+@Entity
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
+public class Ticket {
+
+    @Id
+    @Column(nullable = false)
+    private String type;
```

Comment:
Dlaczego string, a nie enum?

Reply:
Wyszedłem z założenia, że ktoś - nie programista - mógłby chcieć dodać jakieś typy biletów (np. senior) albo nawet zmienić nazwę typu, to by można by to zrobić z poziomu bazy danych, a nie robić zmian w kodzie.